docs/source/backends.rst

```diff
@@ -45,6 +45,34 @@ torch.backends.cuda
 
         Clears the cuFFT plan cache.
 
+.. attribute::  torch.backends.cuda.linalg.preferred_library
```

Comment:
The PR description talks about `torch.backends.linalg.preferred` while this one is under `cuda`. Is that on purpose?

Reply:
It was suggested here https://github.com/pytorch/pytorch/pull/67980#discussion_r751801464 to use `cuda` namespace. I forgot to update the PR description after the code change.